engine/src/main/java/org/terasology/rendering/world/WorldRendererImpl.java

```diff
@@ -169,11 +169,8 @@ public WorldRendererImpl(Context context, GLBufferPool bufferPool) {
     // TODO: one day the main light (sun/moon) should be just another light in the scene.
     private void initMainDirectionalLight() {
         mainDirectionalLight.lightType = LightComponent.LightType.DIRECTIONAL;
-        mainDirectionalLight.lightColorAmbient = new Vector3f(1.0f, 1.0f, 1.0f);
-        mainDirectionalLight.lightColorDiffuse = new Vector3f(1.0f, 1.0f, 1.0f);
-        mainDirectionalLight.lightAmbientIntensity = 1.0f;
-        mainDirectionalLight.lightDiffuseIntensity = 2.0f;
-        mainDirectionalLight.lightSpecularIntensity = 0.0f;
```

Comment:
With the default value of 0.1 and with the fixes,  we can now get specular light coming off the sun/moon.  It is neat because the specular light comes in line with the water specular light. :)

After (specular intensity of 0.1):
![terasology-160408112602-1280x720](https://cloud.githubusercontent.com/assets/5922109/14396133/abfa824e-fd8a-11e5-8901-3362f2df7415.jpg)

After (specular intensity of 0.5 so that you can see what to look for):
![terasology-160408112615-1280x720](https://cloud.githubusercontent.com/assets/5922109/14396136/b854a2b8-fd8a-11e5-824b-082d4988bd03.jpg)